src/main/java/ch/njol/skript/events/EvtPlayerInput.java

```diff
@@ -0,0 +1,54 @@
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerInputEvent;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.InputKey;
+
+import java.util.Set;
+
+public class EvtPlayerInput extends SkriptEvent {
+
+	static {
+		if (Skript.classExists("org.bukkit.Input")) {
+			Skript.registerEvent("Player Input", EvtPlayerInput.class, PlayerInputEvent.class,
+					"player input",
+					"[player] press[ing] %inputkeys%")
```

Comment:
```suggestion
					"[player] press[ing] [of] %inputkeys%")
```
This could also be used for release, right? 
maybe have a press, release, and toggle option?

Reply:
ah true, i didn't think about that